### DISCUSSION TITLE
Store

### DISCUSSION THREAD

**Discussion by Alex-MRX-777** [btcpayserver/btcpayserver#4439] [Technical Support]
When updating store data (various general and checkout tabs, etc.), you need to specify all store settings, otherwise an error will occur, is it possible to make different requests update the general, checkout, etc. tabs and they update the data that relate to these tabs .

**Comment by bolatovumar** [comment]
@Alex-MRX-777 this is about Greenfield API?

**Reply by Alex-MRX-777** [reply]
Yes

**Reply by bolatovumar** [reply]
@Kukks @NicolasDorier `/api/v1/stores/{storeId}` was made a `PUT` instead of `PATCH` for reason, right? Any reason we couldn't also provide a `PATCH` which would allow specifying just some of the things instead of all properties of a store?
